src/libstd/fs.rs

```diff
@@ -1991,6 +1997,28 @@ pub fn remove_dir_all<P: AsRef<Path>>(path: P) -> io::Result<()> {
 ///     Ok(())
 /// }
 /// ```
+///
+/// ```rust,no_run
+/// use std::{fs, io};
+///
+/// fn main() -> io::Result<()> {
+///     // The order read_dir returns entries is not guaranteed. If reproducible
```

Comment:
Minor grammatical error here. Maybe you meant:

> The order _in which_ \`read_dir\` returns entries is not guaranteed.

Reply:
Backticks aren't processed in comments but IMO it still helps visually demarcate that you're talking about a Rust item. I'm not sure if that fits the rust-lang code style. @Centril?

Reply:
Yeah we prefer backticks when referring to code and code entities.